GetComponents

```diff
@@ -1376,7 +1372,7 @@ sub handle_git {
         }
         print_checkout_info( $checkout, $url, $target, $name );
 
-        run_command("mkdir -p $orig_dir/$ROOT/repos");
+        run_command("mkdir -p $ROOT/repos");
```

Comment:
If $ROOT is set to cwd(), wouldn't this pose a similar problem with whitespace? Granted, ROOT is usually set to something by the thornlist, but it might not be, and would be cwd() as of line 387.